src/mmw/sass/pages/_data-catalog.scss

```diff
@@ -33,9 +29,38 @@
         left: 10px;
         font-size: 18px;
     }
-    > .data-catalog-search-description {
-        color: $ui-grey;
-        font-size: $font-size-h4;
-        margin-left: 0.5rem;
+}
+
+.data-catalog-results {
+    margin-top: 1em;
+    font-size: 0.8em;
+
+    .resource {
+        background: #fff;
+        padding: 15px;
+        margin-bottom: 1px;
+
+        &.expanded {
+            .resource-description {
+                display: inline;
+            }
+            .resource-description-short {
+                display: none;
+            }
+        }
+
+        .resource-description {
+            display: none;
+        }
+
+        .resource-title {
+            font-size: 1em;
```

Comment:
I decided to not make any more changes to the style since the design is still in flux.